sunpy/map/header_helper.py

```diff
@@ -238,17 +238,14 @@ def get_observer_meta(observer, rsun: (u.Mm, None)):
     coordinate : `~astropy.coordinates.BaseCoordinateFrame`
         The coordinate of the observer, must be transformable to Heliographic
         Stonyhurst.
-    rsun : `astropy.units.Quantity`
+    rsun : `astropy.units.Quantity`, optional
```

Comment:
Could you add a note here about the consequences of leaving this as `None`? e.g. "if `rsun` is None, the ``rsun_ref`` and ``rsun_obs`` keys will not be set.

Reply:
👍  done